testsuite/podman_runner/09_build_server_code.sh

```diff
@@ -39,5 +39,5 @@ sudo -i podman exec server bash -c "rctaskomatic restart"
 sudo -i podman exec server bash -c "cp /client/tools/mgr-push/*.py /usr/lib/python3.6/site-packages/rhnpush/"
 sudo -i podman exec server bash -c "cp /client/tools/mgr-push/rhnpushrc /etc/sysconfig/rhn/rhnpushrc"
 
-sudo -i podman exec server bash -c "cd /susemanager-utils/susemanager-sls/; cp -R modules/* /usr/share/susemanager/modules; cp -R salt/* /usr/share/susemanager/salt; cp -R salt-ssh/* /usr/share/susemanager/salt-ssh"
+sudo -i podman exec server bash -c "cd /susemanager-utils/susemanager-sls/; cp -R modules/* /usr/share/susemanager/modules; cp -R salt/* /usr/share/susemanager/salt; cp -R src/modules/* /usr/share/susemanager/salt/_modules; cp -R salt-ssh/* /usr/share/susemanager/salt-ssh"
```

Comment:
when this fix your problem, you may want to add also all the other directories?
At least grains and states. Not sure, but why not also beacons